### PR TITLE
Add getting started section

### PR DESCRIPTION
**Pull request by jackboberg**
Idea from @tzmanics (#10)

### CODE REVIEW

**Comment by jackboberg** [comment]
Any other links/notes we want to add to this section?

**Comment by fiveisprime** [comment]
Can you also add something about gitter - I know there's a link right above as
part of the badge, but that doens't mean anything to people unfamiliar with
gitter. Something like "Join this chat with questions."

I'd also like a bit about getting help. Like "Stumped on an exercise, ask in
gitter or open an issue." Just a way to let people know that it's totally cool
to open issues and get in touch with any of us.

I'd also like to see "Join us by opening an issue", but I'm not totally sure
how to word that correctly. I want to add people as members and make them feel
welcome; however we put that into words lol.. maybe that could be a section on
its own..

**Comment by fiveisprime** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
 
-[![Join the chat at https://gitter.im/nodeschool/cincinnati](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/nodeschool/cincinnati?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Join the chat at https://gitter.im/nodeschool/cincinnati](https://badges.gitter.im/Join%20Chat.svg)][gitter]
 
 NodeSchool chapter for Cincinnati.
 
+
+## Getting Started
+
+Here is a list of helpful resources if you are new to node school events.
```
Comment (fiveisprime) [inline README.md]: Change "node school" to "NodeSchool" to match how it is throughout the document.

**Comment by tzmanics** [comment]
This is great! Thanks for doing it, Jack!

I think the 'Getting Started' section should be a little more explicit, i.e. "This is what you need to get set up for the workshopper -Node installed -npm updated -a text editor you're comfortable with" or something along those lines. 

My thought was for a page that walks attendees who aren't set up through installation and prep so they don't have to wait for a mentor to come around (or can set up ahead of time) and can get into the workshopper ASAP. Also, that it gets the attendee to our github page and then they have it in their history if they want to comment, reference or become a member. (👈 this feels very important)

Then we can follow with the 'Other helpful resources' section. I think this page may change or expand to a whole resources section once we assess different needs from different workshops.

woooooooo!

**Comment by jackboberg** [comment]
Took a look at other chapters README's, and most are pretty sparse. Here is a more full-featured one: https://github.com/nodeschool/dallas

**Comment by fiveisprime** [comment]
Oh, I love the attendance section!

**Comment by jackboberg** [comment]
@tzmanics Can you maybe provide as basic outline of your ideas for getting started content? Maybe we can develop that as a separate document for now, and then link or inline once we like what's produced there.

**Comment by tzmanics** [comment]
I thought we decided this morning that this isn't something we wanted to orchestrate since there are so many approaches for a user to use/consider.

**Comment by jackboberg** [comment]
No reason we can't still try to develop some of your nice step-by-step instruction sets where we can. I was only concerned they won't fit well on the 'intro' README. I think a new issue/branch...

**Comment by fiveisprime** [comment]
I really like the state of this PR. I think a new PR for figuring out how deep
we want to go with a how-to style getting started guide is a great idea.

**Comment by tzmanics** [comment]
Sounds good to me 😊👍
I feel like there are other things we may want to concentrate on first.

**Comment by ghost** [comment]
Are we OS agnostic?

**Comment by fiveisprime** [comment]
I'm not sure what you mean here. The workshoppers run on Node.js, so.. yeah?

**Pull request merged**